pom.xml

```diff
@@ -28,9 +32,10 @@
 
   <dependencies>
     <dependency>
-      <groupId>org.bouncycastle</groupId>
-      <artifactId>bcpkix-jdk15on</artifactId>
-      <version>1.54</version>
+       <groupId>org.jenkins-ci.plugins</groupId>
+       <artifactId>bouncycastle-api</artifactId>
```

Comment:
Only in the tests to make sure that what we read and write is consistent with what BC would do:
https://github.com/jenkinsci/instance-identity-module/pull/6/files#diff-12d629d9d550350a15b7ba219656ba62R140